toast-tk-plugin-loader/src/main/java/io/toast/tk/agent/config/AgentConfigProvider.java

```diff
@@ -0,0 +1,69 @@
+package io.toast.tk.agent.config;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.google.inject.Provider;
+
+public class AgentConfigProvider implements Provider<AgentConfig> {
+	
+	private static final Logger LOG = LogManager.getLogger(AgentConfigProvider.class);
+
+	private AgentConfig webConfig;
+	
+	public static final String TOAST_TEST_WEB_INIT_RECORDING_URL = "toast.web.recording.url";
+
+	public static final String TOAST_CHROMEDRIVER_PATH = "toast.chromedriver.path";
+	
+	public static final String TOAST_TEST_WEB_APP_URL = "toast.webapp.url";
+
+	public static final String TOAST_API_KEY = "toast.api.key";
+	
+	public static final String TOAST_SCRIPTS_DIR = "toast.agent.scripts.dir";
+	
+	public static final String TOAST_PLUGIN_DIR = "toast.plugin.dir";
+
+	public static final String TOAST_PROXY_ACTIVATE = "toast.proxy.activate";
+	
+	public static final String TOAST_PROXY_ADRESS = "toast.proxy.adress";
+
+	public static final String TOAST_PROXY_PORT = "toast.proxy.port";
+
+	public static final String TOAST_PROXY_USER_NAME = "toast.proxy.username";
+
+	public static final String TOAST_PROXY_USER_PSWD = "toast.proxy.userpswd";
+
+	private void initConfig() {
+		String userHomepath = AgentConfig.getToastHome() + "/";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this hard-coded path-delimiter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1075)